File: user_tests/bubble_sort/bubble_sort.jl
function bubble_sort(seq::Vector{Int64})::Vector{Int64}
    l = length(seq)
    for _ = 0:l-1
        for n = 1:l-1
            if seq[n+1] < seq[n]
                seq[n], seq[n+1] = (seq[n+1], seq[n])
            end
        end
    end
    return seq
end

File: user_tests/composite/bubble_sort.py
def bubble_sort(seq: List[int]) -> List[int]:
    l = len(seq)
    for _ in range(l):
        for n in range(1, l):
            if seq[n] < seq[n - 1]:
                seq[n - 1], seq[n] = seq[n], seq[n - 1]
    return seq